libpod/common/common.go

```diff
@@ -69,15 +59,6 @@ func IsValidBool(str string) bool {
 	return IsTrue(str) || IsFalse(str)
 }
 
-// GetPolicyContext creates a signature policy context for the given signature policy path
-func GetPolicyContext(path string) (*signature.PolicyContext, error) {
-	policy, err := signature.DefaultPolicy(&types.SystemContext{SignaturePolicyPath: path})
-	if err != nil {
-		return nil, err
-	}
-	return signature.NewPolicyContext(policy)
-}
-
```

Comment:
These two functions may not be used ATM, but I"m always hesitant to remove tool like functions that someone found useful at some point in time.